cache/src/main/java/org/buffer/android/boilerplate/cache/db/constants/ArticleConstants.kt

```diff
@@ -0,0 +1,14 @@
+package org.buffer.android.boilerplate.cache.db.constants
+
+/**
+ * Defines constants for the Article Table
+ */
+object ArticleConstants {
```

Comment:
Wait, is there a reason why using `object` constant here? @adesamp 

See here about constant: 

https://sampingan.atlassian.net/wiki/spaces/S/pages/38141970/Android+Code+Convention#Writing-meaningful-%E2%80%9CConstant%E2%80%9D